vumi/persist/tests/test_txriak_manager.py

```diff
@@ -217,19 +206,22 @@ def test_run_riak_map_reduce_with_timeout(self):
             function(value, keyData) {
                 var date = new Date();
                 var curDate = null;
+                // Wait 11ms so we run past the 10ms timeout.
                 do { curDate = new Date(); }
                 while(curDate-date < 11);
+                return value;
             }
             """)
 
         try:
             yield self.manager.run_map_reduce(mr, lambda m, l: None)
         except Exception, err:
-            msg = unrepr_string(str(err))
-            self.assertTrue(msg.startswith(
-                "Error running MapReduce operation."))
-            self.assertTrue(msg.endswith(
-                "Body: '{\"error\":\"timeout\"}'"))
+            msg = str(err)[1:-1].decode("string-escape")
+            if not all([
```

Comment:
I find the `all()` style a little cleaner than `(foo and bar)` when the expression spans multiple lines. The short-circuiting of the `and` operator wouldn't buy much here, so it basically just boils down to a mild readability preference. Both forms work and I'm happy to change it if you think the `and` works better -- I'm never really sure whether this kind of readability preference is a personal quirk of mine or something more general.